package.json

```diff
@@ -284,6 +284,7 @@
 		"@typescript-eslint/eslint-plugin": "1.10.2",
 		"@typescript-eslint/parser": "1.10.2",
 		"@wordpress/babel-plugin-makepot": "2.1.3",
+		"@wordpress/dependency-extraction-webpack-plugin": "^1.0.1",
```

Comment:
Do we need/want it at the top level? Or should we move it to `apps/wpcom-block-editor/package.json`?

Reply:
As a `devDependency`, it has to be at top level.